index.js

```diff
@@ -102,8 +108,20 @@ class MultiCoreIndexer extends TypedEmitter {
     this.#indexStream.addStream(coreIndexStream)
   }
 
+  /**
+   * Resolves when indexing state is 'idle'
+   */
+  async idle() {
```

Comment:
nit but not sure about the naming of this, but can't think of a suggestion either

Reply:
Yeah I often want to call functions like this "awaitIdle", but that just becomes confusing when you write `await awaitIdle()`